src/components/10-atoms/heading/ui.test.js

```diff
@@ -0,0 +1,646 @@
+import { Selector } from 'testcafe';
+
+const host = process.env.TEST_HOST_STORYBOOK_URL;
+
+fixture('Heading - Correct CSS attributes')
+  .page(`${host}/iframe.html?id=atoms-heading--heading`)
+  .afterEach(async t => {
+    await t.maximizeWindow();
+  });
+
+const TAG = 'axa-heading';
+
+const mobileBreakpoint = 767;
+const tabletBreakpoint = 991;
+const desktopBreakpoint = 992;
```

Comment:
breakpoint are global accessible aren't they?

Reply:
Yes, but only in the scss context. We could think about adding a global testfile-variables file though.

Reply:
👍